### ISSUE TITLE
Precompile for frontend use

### ISSUE THREAD

**Issue by antejan** [slm-lang/slm#21]
Hey! Sorry if it's inappropriate place to ask such question. Is there a way to precompile slm templates so it will be possible to use them on frontend? 

We have Rails app with Backbone frontend and slim templates ([skim](https://github.com/appjudo/skim) actually) and we're trying now to move from rails infrastructure to gulp/webpack and slm looks like the only node-based engine we could use without rewriting everything. For example it could look like [this](https://github.com/ericdfields/hamlc-loader).

**Comment by yury** [comment]
Yep, It is possible... That is why vm_browser exists. But it is not tested very well.

**Comment by antejan** [comment]
Thanks for your reply, sounds promising. Maybe you can show an example of using slm this way?
